docs/src/modules/components/overview/Customization.tsx

```diff
@@ -0,0 +1,389 @@
+              <StaticDatePicker
+                defaultValue={dayjs('2022-04-17')}
+                slotProps={{ layout: { sx: { minWidth: 'initial' } } }}
+              />
+            </Card>
+          )}
+        </ThemeProvider>
+      </Stack>
+    </React.Fragment>
+  );
+}
+
+const initialState: Config = {
+  selectedTheme: 'custom',
+  color: 'default',
+  layout: 'horizontal',
+  density: 'medium',
+  corner: 'medium',
+  typography: 'default',
+};
+
+export default function Customization() {
+  const [styleConfig, setStyleConfig] = React.useState<Config>(initialState);
+
+  const brandingTheme = useTheme();
+
+  const colorScheme = getColorScheme(styleConfig.selectedTheme);
+
+  const handleChangeTheme = (_event: React.MouseEvent<HTMLElement>, newTheme: Themes) => {
+    setStyleConfig((prev) => ({
+      ...prev,
+      selectedTheme: newTheme,
+      corner: newTheme === 'custom' ? 'medium' : 'rounded',
+    }));
+  };
+  const handleChangeColor = (_event: React.MouseEvent<HTMLElement>, newColor: string) => {
+    setStyleConfig((prev) => ({ ...prev, color: newColor }));
+  };
+  const handleChangeLayout = (_event: React.MouseEvent<HTMLElement>, newLayout: Layout) => {
+    setStyleConfig((prev) => ({ ...prev, layout: newLayout }));
+  };
+  const handleChangeCorner = (_event: React.MouseEvent<HTMLElement>, newCorner: Corner) => {
+    setStyleConfig((prev) => ({ ...prev, corner: newCorner }));
+  };
+  const handleChangeDensity = (_event: React.MouseEvent<HTMLElement>, newDensity: Density) => {
+    setStyleConfig((prev) => ({ ...prev, density: newDensity }));
+  };
+
+  const handleChangeTypography = (
+    _event: React.MouseEvent<HTMLElement>,
+    newTypography: TypographyType,
+  ) => {
+    setStyleConfig((prev) => ({ ...prev, typography: newTypography }));
+  };
+
+  return (
+    <React.Fragment>
+      <Divider />
+      <Stack spacing={4} py={8} alignItems="center">
+        <Stack spacing={1} sx={{ maxWidth: { xs: '500px', md: '100%' } }}>
+          <Typography variant="body2" color="primary" fontWeight="semiBold" textAlign="center">
+            Customization
+          </Typography>
+          <Typography
+            variant="h4"
+            component="h2"
+            fontWeight="semiBold"
+            color="text.primary"
+            textAlign="center"
+          >
+            Highly customizable components
+          </Typography>
+          <Typography variant="body1" color="text.secondary" textAlign="center">
+            Easily adaptable to any style, our components leverage Material Design for
+            out-of-the-box elegance and support extensive customization to perfectly align with your
+            branding.
+          </Typography>
+
+          <Stack
+            maxWidth="xl"
+            direction={{ xs: 'column', md: 'row' }}
+            sx={{
+              display: 'flex',
+              border: '1px solid',
+              borderColor: 'divider',
+              borderRadius: 1,
```

Comment:
This element could use some more spacing above it to match the other sections. On the parent `Stack`, if you switch the `spacing` prop to `gap`, you could use `marginTop`/`mt` here without the spacing value overriding it.